pkgs/development/compilers/dasm/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub}:
```

Comment:
You don't consider my suggestions nits? Well then, here's a nit for ya:

```suggestion
{ stdenv, fetchFromGitHub }:
```

(please :) )

Reply:
lol. Nope! bring it on :)

ahh. I just discovered `nixfmt`, too.

Reply:
Yeah, nixfmt is cool but it's not official yet. I.e. it's not the standard for nixpkgs.